src/core/brand/SvgRoundelBrand.tsx

```diff
@@ -0,0 +1,17 @@
+import { SvgRoundel } from './SvgRoundel';
+import { brand } from '@guardian/src-foundations/palette';
+
+export type SvgRoundelBrandProps = {
+	/**
+	 * The width the SVG will display at (height is automatically adjusted
+	 * to maintain the correct aspect ratio).
+	 */
+	width?: number;
+};
```

Comment:
Probably one for future PR but I think we should abstract this into a shared type to save some duplication

Reply:
i did actually do that originally, but wasn't sure if it was better to be explicit per component? easy either way